data-source/census_individual.jsonnet

```diff
@@ -111,11 +111,7 @@ local confirmation = import 'blocks/confirmation.json';
       validator: 'optional_string',
     },
     {
-      name: 'address-line-1',
-      validator: 'string',
-    },
-    {
-      name: 'address-line-2',
+      name: 'display-address',
```

Comment:
This should be `display_address`. I don't know why `address-line-1` and `address-line-2` were different, but all of the other metadata uses underscores rather than hyphens.

Reply:
done